File: 05-Problem-Variables/05-Problem-Variables-CPP/05-Problem-Variables.cpp
// Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

#include <iostream>
using namespace std;

int main() {

    int myMutable = 5;
    const int immutable = 6;

    cout << "mutable: " << myMutable << " immutable: " << immutable << endl;

    myMutable += immutable;

    cout << "mutable: " << myMutable << " immutable: " << immutable << endl;

  return 0;
}

File: 05-Problem-Variables/05-Problem-Variables.swift
// Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

var mutable = 5;
let immutable = 6;

print("mutable: \(mutable) immutable: \(immutable)");

mutable += immutable;

print("mutable: \(mutable) immutable: \(immutable)");
